src/main/java/HttpServer/MockClient.java

```diff
@@ -0,0 +1,26 @@
+package HttpServer;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class MockClient implements Writable {
```

Comment:
Mocks/Fakes typically live in the test directory.

Reply:
:white_check_mark: